zebra-state/src/service/non_finalized_state/chain.rs

```diff
@@ -339,6 +354,14 @@ impl Chain {
             )?;
         }
 
+        let rebuild_time = start_time.elapsed();
+        info!(
+            ?rebuild_time,
+            ?fork_height,
+            ?fork_tip,
+            "finished rebuilding note commitment trees after a non-finalized chain fork",
+        );
```

Comment:
Optional: it would be nice to log the number of blocks after the fork, to have a notion about how much time it takes per block

Reply:
I'll try to push a quick change

Reply:
I logged the number of rebuilt blocks and the time per block in 1898f7ae9

Reply:
Ah, feel free to ignore this, I just realized we rebuild the entire non-finalized chain so it's not dependent of the fork length